src/seamless_communication/cli/streaming/evaluate.py

```diff
@@ -69,6 +79,11 @@ def main() -> None:
 
         if args.expressive:
             agent_class = SeamlessS2STAgent
+            if not args.gated_model_dir:
+                raise ValueError(
+                    "--gated-model-dir must be provided for the Seamless model."
+                )
+            add_gated_assets(args.gated_model_dir)
```

Comment:
We do `add_gated_assets` [here](https://github.com/fairinternal/seamless_communication/blob/e430d557c6232c214ba4fc55b2c3a129a846af96/src/seamless_communication/streaming/agents/pretssel_vocoder.py#L37)  anyways, so how is seamless inference broken?

Reply:
You are correct, I reverted the change. I forgot to add --gated-model-dir to the streaming_evaluate CLI, I've made the change now in the README.